net/kafkacat/Portfile

```diff
@@ -11,12 +12,13 @@ maintainers         {@alexeyt820 gmail.com:alexey.trenikhin+macports} openmainta
 description         Generic command line non-JVM Apache Kafka producer and consumer
 long_description    Generic non-JVM producer and consumer for Apache Kafka >=0.8, think of it as a netcat for Kafka.
 
-checksums           sha256 bbbb60e5f591f896c44246e8fb4dd13f7974f1032a3404e8e9fdc77865293bfb \
-                    rmd160 dba7019d9da4e74dd722fcef3aa1e54eb8174a0f \
-                    size 124695 
+replaced_by         kcat
+revision            1
+livecheck.type      none
 
-configure.args      --enable-json
-
-depends_lib         port:yajl \
-                    port:librdkafka
+pre-configure {
+    ui_error "Please do not install this port since it has been replaced by 'kcat'."
```

Comment:
There's nothing to install. You should instead simply inform the user that the port has been replaced by kcat. But I believe the obsolete portgroup already does something like that.

Reply:
Indeed, see the guide how to use the "obsolete" PortGroup to replace a port. Additionally, please add a comment when the obsokete port can be removed (i.e., 1 year from now).

Reply:
Thanks for the guidelines, I've made the changes.